### ISSUE TITLE
Request to restore crhook from PR #277 as a 4th option or roll back

### ISSUE THREAD

**Issue by cbirkeland** [QW-Group/ktx#303]
Current CTF community has been split between the old hook mode 'crhook' and the three new votable hook modes. CTF admins would really appreciate crhook being restored as a 4th option, and keeping the new implemented hook types. 

Changes should be found in grapple.c . 
To summarize, the difference between crhook and hookfast is 1200 to 1050. It's just the 1200 velocity we would like to have an option to pick. The changes to the sound are great and appreciated.

If it isn't doable to restore crhook and make it a 4th option. The CTF admins would like to roll back the PR #277 

![Screenshot_20230707-104752](https://github.com/QW-Group/ktx/assets/42343590/33fed637-00c1-4224-8e6d-3b45405aa96c)

**Comment by tcsabina** [comment]
@inf1niti,
How do you see this?

**Comment by inf1niti** [comment]
I don't think we should rollback PR #277 because, aside from adding votable hook modes, it adds the following bugfixes and QoL improvements:

> - improve hook & rune sounds (specify channels, jump no longer cancels rune sounds)
> - remove annoying looped grapple sounds for simpler axe swish(throw) and axe hit (anchor) sounds
> - improve balance of haste-hook implementation; make grapple speed a factor of haste power
> - fixes chain clear bug & improves hook animation smoothness

To add a new hook mode to the votable implementation is pretty easy.  You can follow the boilerplate in the [diff for PR #277](https://github.com/QW-Group/ktx/pull/277/files).  There's quite a bit of boilerplate, but it should be easy enough to follow and copy/pasta the patterns for adding new options to the existing vote. 

When you've added the voting option, you need to add some code in [grapple.c](https://github.com/QW-Group/ktx/blob/master/src/grapple.c):

- line ~14:  add a new throw speed definition (1200)
- line ~432: set the appropriate speed definition for the new hookstyle
- line 455-460: you might want to ignore the hasteMultiplier for hookspeed 1200, as it's going to make the hook throw even faster (multiplies throw speed by 1.125% - so would be 1350 😬)

If any other help or guidance is needed, feel free to ping me on discord.  I'm available to review a PR when you have something to post.  👍
